.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ commands:
     steps:
       - run:
           name: "Check import order with isort"
-          command: isort --check-only
+          command: isort --check-only -v
```

Comment:
Switching this to verbose to have more details on incorrect orderings in CircleCI